pom.xml

```diff
@@ -13,5 +13,12 @@
         <maven.compiler.target>19</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
In EdHub en de backend les gebruikten we junit-jupiter-api. Gebruik je bewust junit-jupiter-engine?